src/main/java/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,35 @@
+package pages;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import utils.WaitActions;
+
+public class DifferentElementsPage {
+
+    @FindBy(xpath = "//label[text()[contains(., ' Water')]]/*[@type='checkbox']")
```

Comment:
Why do you use public modifier for your web elements? Please, make it all private.